lib/hydra/works/models/collection.rb

```diff
@@ -0,0 +1,6 @@
+module Hydra::Works
+  class Collection < ActiveFedora::Base
+    include Hydra::PCDM::CollectionBehavior
```

Comment:
Hydra::PCDM::CollectionBehavior is a dependency of Hydra::Works::CollectionBehavior, so
`include Hydra::PCDM::CollectionBehavior` should be called within the `include` block of Hydra::Works::CollectionBehavior.